.github/workflows/update-fetchgit.yml

```diff
@@ -0,0 +1,95 @@
+name: Update fetchGit repositories
+
+on:
+  push:
+
+  pull_request:
+
+  workflow_dispatch:
+
+  schedule:
+    # run three times a day
+    - cron: "0 4,12,20 * * *"
+
+jobs:
+  update-nix-fetchgit:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # for git push
+      contents: write
+      # to allow opening the PR
+      pull-requests: write
+
+    env:
+      BRANCH_NAME: "update-fetchgit"
+
+    steps:
+      - name: Install Nix
+        uses: cachix/install-nix-action@v25
+        with:
+          # needed at least for act
+          # TODO: I have no idea how to properly fill this variable... sigh...
+          #nix_path: nixpkgs=channel:nixpkgs-unstable
+          # avoids rate limits
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This shouldn't be necessary, AFAIR this variable is used by default.

Reply:
It's needed for the act runner which otherwise runs into the rate limit. Actions cannot automatically use the token, it must be passed explicitly (security feature). Obviously, GitHub's own runners are exempt from the rate limit.

Reply:
I mean, we're using the GitHub-hosted runners right now, so it would work. Otherwise we'd need the same workaround in our other files.

Reply:
The other workflow can't really be tested using `act` since it'll purge your entire local Docker image cache...

Reply:
Please specify that this is required only for `act`, then I'm fine with the change.